portfolio/src/main/webapp/script.js

```diff
@@ -83,7 +85,8 @@ function createCommentElement(comment) {
  */
 function init() {
   document.getElementById("background-button").addEventListener("click", setRandomBackgroundColor);
-  document.getElementById("comment-button").addEventListener("click", loadComments);
+  document.getElementById("num-comments").onload = loadComments();
```

Comment:
Just wondering if this line is necessary... does calling `loadComments()` during `onload` actually load in any comments? I don't think `num-comments` would be set to anything other than 0 initially, right?

Reply:
The default value for the selector is 1, so it will have 1 comment when it loads the page